Gemfile

```diff
@@ -113,6 +113,9 @@ group :development, :test do
   gem 'i18n_generators'
 
   gem 'sprockets', '~>2.12.3' # spckets 3.0.3 breaks konacha
+
+  # Mocks and stubs for javascript tests
+  gem 'sinon-rails'
```

Comment:
Changed my mind. Adding the script to the vendor is still not good enough. The right fix is trying to reach the gem's maintainer and, if he is not going to update, fork it, release and maintain it.

(I intend to do that https://github.com/travisjeffery/sinon-rails/issues/9)

Lets keep the gem for now and, after the closing of this, we update. Are we ok with this?